sci-libs/rpp/rpp-6.1.1.ebuild

```diff
@@ -54,7 +57,10 @@ src_prepare() {
 		-i CMakeLists.txt || die
 
 	cmake_src_prepare
-	use test && rcc_test_wrapper cmake_src_prepare
+	if use test; then
+		local PATCHES=()
+		rcc_test_wrapper cmake_src_prepare
+	fi
```

Comment:
I think this would be cleaner:
```
if use test; then
    rcc_test_wrapper cmake_src_prepare
else
    cmake_src_prepare
fi
```

Reply:
@AndrewAmmerlaan , no, it won't work. I have to run cmake_src_prepare twice, first time in build directory, second time in nested directory for tests (it will create `gentoo_rules.cmake`). Then cmake_src_configure twice, then cmake_src_compile twice. Tests are done as independent cmake project there.

Reply:
Aah I see, then this looks good to me :+1: